test/cctest/test_node_postmortem_metadata.cc

```diff
@@ -0,0 +1,138 @@
+// This file is a template used by ./tools/gen-postmortem-metadata.py to
+// generate a proper test file. The comment below is replaced by
+// `tools/gen-postmortem-metadata.py` to add the constants generated by it.
+
+// {{debug_symbols_file}}
+
+
+#include "gtest/gtest.h"
+#include "node.h"
+#include "node_internals.h"
+#include "node_test_fixture.h"
+#include "req_wrap-inl.h"
+#include "tracing/agent.h"
+#include "v8.h"
+
+const int node::Environment::kContextEmbedderDataIndex;
```

Comment:
Without this line, the Debug build breaks, `make -j4 BUILDTYPE=Debug` throws the following error:

```
/home/matheus/workspace/sthima/node/out/Debug/obj.target/cctest/test/cctest/test_node_postmortem_metadata.o: In function `DebugSymbolsTest_ContextEmbedderDataIndex_Test::TestBody()':
/home/matheus/workspace/sthima/node/out/../test/cctest/test_node_postmortem_metadata.cc:46: undefined reference to `node::Environment::kContextEmbedderDataIndex'
collect2: error: ld returned 1 exit status
cctest.target.mk:217: recipe for target '/home/matheus/workspace/sthima/node/out/Debug/cctest' failed
make[1]: *** [/home/matheus/workspace/sthima/node/out/Debug/cctest] Error 1
```

Which is weird, because it works on `node_postmortem_metadata.cc`. Suggestions?

Reply:
Neither of them worked :(
Debug build still gives the same error

Reply:
I guess the linker is complaining because`kContextEmbedderDataIndex` is ODR-used below, you can make it not ODR-used by applying lvalue-to-rvalue conversion immediately on it (e.g. assigning it to a `int` before using it in `EXPECT_EQ`).

Reply:
Worked, thanks! :tada:

```c++
TEST_F(DebugSymbolsTest, ContextEmbedderDataIndex) {
  int kContextEmbedderDataIndex = node::Environment::kContextEmbedderDataIndex;
  EXPECT_EQ(nodedbg_const_Environment__kContextEmbedderDataIndex__int,
            kContextEmbedderDataIndex);
}
```